fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -388,10 +388,18 @@ public interface Count extends StoreTimer.Event {
     }
 
     /**
-     * Contains the number of occurrences and cummulative time spent on an associated {@link StoreTimer.Event}.
+     * {@link Event}s that count the number of occurrences of an operation and also the size of the operation.
+     * There is no meaningful time duration associated with these events.
```

Comment:
I think there should probably be something here to explain why someone would want to use this over, say, a `Count`--particularly how this differs from `Count` events where `isSize()` is true.

Reply:
I am not quite sure why `isSize()` is used and was introduced. It seems to be there just to distinguish "true" Count events from "size" counts in `CountAggregate` event but that doesn't seem to be a sufficient use case.

Reply:
Added a brief description here